src/ui/marker.js

```diff
@@ -604,4 +607,29 @@ export default class Marker extends Evented {
     getPitchAlignment() {
         return this._pitchAlignment;
     }
+
+    /**
+     * Sets the current `color` property of the marker.
+     * @param {string} [color] Sets the `color` property of the marker. If alignment is 'auto', it will automatically match `rotationAlignment`.
+     * @returns {Marker} `this`
+     */
+    setColor(color: ?string) {
+        if (color) {
+            this._color = color;
+        } else {
+            this._color = this._defaultColor;
+        }
+
+        this._background.setAttributeNS(null, 'fill', this._color);
+
+        return this;
+    }
+
+    /**
+     * Returns the current `color` property of the marker.
+     * @returns {string} The current color of the marker.
+     */
+    getColor() {
```

Comment:
I think we should specify what should happen if getColor is called when the Marker isn't the default marker element. I'd probably say in that case it should return undefined, but open to other opinions.

Reply:
I’m not particular about it. I think so too as you said. 😊